native/cocos/renderer/pipeline/custom/RenderCommonJsb.h

```diff
@@ -52,15 +46,6 @@ bool nativevalue_to_se(const cc::render::MovePair &from, se::Value &to, se::Obje
 template <>
 bool sevalue_to_native(const se::Value &from, cc::render::LightInfo *to, se::Object *ctx); // NOLINT
 
-template <>
-bool sevalue_to_native(const se::Value &from, cc::render::Descriptor *to, se::Object *ctx); // NOLINT
-
-template <>
-bool sevalue_to_native(const se::Value &from, cc::render::DescriptorBlockFlattened *to, se::Object *ctx); // NOLINT
-
-template <>
-bool sevalue_to_native(const se::Value &from, cc::render::DescriptorBlockIndex *to, se::Object *ctx); // NOLINT
```

Comment:
Doesn't need these conversions now ?

Reply:
I think they are not needed now. These values are filled by serialization now.